CMakeLists.txt

```diff
@@ -323,7 +323,11 @@ If(Geant3_FOUND)
  add_subdirectory (geane)
  add_subdirectory (trackbase)
 EndIf ()
-add_subdirectory (eventdisplay)
+
+execute_process(COMMAND ${ROOTSYS}/bin/root-config --has-opengl OUTPUT_VARIABLE ROOT_HAS_OPENGL)
+If (${ROOT_HAS_OPENGL} MATCHES "yes")
+  add_subdirectory (eventdisplay)
+EndIf ()
```

Comment:
If you want to be nice to the user, you could add an else case with `message(STATUS "Not building the eventdisplay because ROOT was built w/o OpenGL support")` or similar.

Reply:
also done